packages/grafana-ui/scripts/generate-icon-bundle.js

```diff
@@ -0,0 +1,50 @@
+const fs = require('fs');
```

Comment:
Aims to replace https://github.com/grafana/icons/blob/main/cmd/genbundle/main.go